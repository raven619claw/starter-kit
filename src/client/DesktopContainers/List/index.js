import React, { Component } from 'react'

// eslint-disable-next-line react/prefer-stateless-function
export default class List extends Component {
  render() {
    return <div>List Desktop</div>
  }
}
