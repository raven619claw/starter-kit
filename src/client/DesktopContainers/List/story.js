import React from 'react'

export default {
  title: 'List'
}

export const List = () => <div>list story</div>
