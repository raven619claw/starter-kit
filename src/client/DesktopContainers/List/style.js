import { css } from '@emotion/core'

export const style = css`
  background: red;
`
