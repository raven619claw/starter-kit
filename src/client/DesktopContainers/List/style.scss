.class1 {
  color: blue;
}
