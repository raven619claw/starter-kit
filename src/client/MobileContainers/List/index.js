import React from 'react'
import ContainerHOC from 'client/CommonComponents/ContainerHOC'

const ListContainerMobile = () => <div className="class1">List Mobile</div>

export default ContainerHOC(ListContainerMobile)
