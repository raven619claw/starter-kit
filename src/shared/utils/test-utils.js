import { render } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
// add the theme and translations object and store here
const AllTheProviders = ({ children }) => children

const customRender = (ui, options) =>
  render(ui, { wrapper: AllTheProviders, ...options })

// re-export everything
export * from '@testing-library/react'

// override render method
export { customRender as render }
