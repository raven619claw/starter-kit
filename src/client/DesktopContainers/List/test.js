import React from 'react'
import { render, cleanup } from 'test-utils'
import List from './index'

afterEach(cleanup)

test('matches List route snapshot', () => {
  const { asFragment } = render(<List />)
  expect(asFragment()).toMatchSnapshot()
})
