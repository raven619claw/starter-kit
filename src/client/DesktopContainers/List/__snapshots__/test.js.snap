// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches List route snapshot 1`] = `
<DocumentFragment>
  .emotion-0 {
  background: red;
}

<div
    class="class1 emotion-0"
  >
    List Desktop
  </div>
</DocumentFragment>
`;
